### DISCUSSION TITLE
Zahl 5 statt 3 - wird nicht richtig erkannt

### DISCUSSION THREAD

**Discussion by surger13** [jomjol/AI-on-the-edge-device#3128] [General] [answered]
Hallo zusammen,
ich habe mir das System ganz neu für meinen Stromzähler aufgesetzt und bin begeistert, was es nach anfänglichen Schwierigkeiten leistet. Kompliment dafür.

Ein Problem ist gerade nur die Erkennung der 4. Stelle. Dort wird statt einer 5 eine 3 erkannt, obwohl sich die 3 eigentlich deutlich unterscheidet (vgl. 1. Stelle) und die 5 auch bei der letzten Stelle 5.9 erkannt wird. 

Hat jemand eine Idee was hier zu ändern bzw. einzustellen wäre? 

![20240704_01](https://github.com/jomjol/AI-on-the-edge-device/assets/35420239/15006d52-a2f7-4aa2-91f4-7677ce6953bb)

Danke

**Comment by SybexX** [comment]
Dein Bild ist unscharf und verschwommen, versuche mal die Linse besser zu fokussieren oder spiel mit den Kameraeinstellungen rum um eine Verbesserung zu erzielen. Zwischischen der Zahl und dem Hintergrund sollte am besten eine klare Farbtrennung vorhanden sein.
Was eventuell auch helfen könnte, ist das model zu wechseln https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-model

**Reply by surger13** [reply, marked as answer]
Danke für die Hinweise. 
Nach ein paar minimalen Verbesserungen durch nochmalige Fokussierung der Cam, wurde nach ein paar anfänglich richtigen Erkennungen als 5, wieder fortlaufend eine 3 ermittelt.
Abhilfe brachte dann schlussendlich das Model auf dig-class100 zu stellen. Jetzt läuft es aktuell wie es soll.

![20240705_01](https://github.com/jomjol/AI-on-the-edge-device/assets/35420239/5f3e9e53-b46b-429b-9dee-e4b981d10e8b)
